### ISSUE TITLE
Cannot generate web app

### ISSUE THREAD

**Issue by julian-martinez-del-campo** [nstudio/xplat#78]
After following the install instructions, when I try to generate an app with the angular-cli I get the following error: ```Path "/apps/web-MyApp/src/index.html" does not exist```.

The generation of a nativescript application seems to work without any problems.

Repreduction steps:
- npm install -g @nrwl/schematics
- npm install -g @angular/cli
- create-nx-workspace MyWorkspace
- npm install @nstudio/schematics --save-dev
- ng g xplat --prefix=myprefix --platforms=web,nativescript
- ng g app MyApp

Outputs:
```
ng g app MyApp
? In which directory should the application be generated? apps
? Would you like to add Angular routing? Yes
? Which Unit Test Runner would you like to use for the application? Karma [ https://karma-runner.github.io ]
? Which E2E Test Runner would you like to use for the application? Protractor [ https://www.protractortest.org ]
? Which tags would you like to add to the application? (used for linting) 
Path "/apps/web-MyApp/src/index.html" does not exist.
```

**Comment by NathanWalker** [comment]
Thank you for reporting this. Looks like could be related to first and second prompt.

**Comment by NathanWalker** [comment]
@julian-martinez-del-campo this is fixed in `7.2.8` now - if you reinstall dependencies it will pull down latest and the web app generator should work with above naming. It was related to capitalization actually `MyApp` - however you can stick with same caps and name will be properly sanitized.

**Comment by julian-martinez-del-campo** [comment]
@NathanWalker Excellent! Thank you so much for the quick fix. I would have never guessed it had to do with the capitalization.
